mkdocs.yml

```diff
@@ -225,6 +225,8 @@ nav:
     - Lamp of Malice:
       - Text Guide: './events/lamp-of-malice/lamp-of-malice-text-guide.md'
       - Video Guide: './events/lamp-of-malice/lamp-of-malice-video-guide.md'
+    - Old Castle Ruins:
+      - Maps and Text Guide: './events/lamp-of-malice/old-castle-ruins-guide.md'
```

Comment:
Instead of putting this in the `lamp-of-malice` folder, can you create a new folder called `old-castle-ruins` and move it to that? I figure we can have a subfolder under `events` for each event that comes up, and each event's subfolder would have its own img directory with the images specific to that event!